src/app/modules/bounce/__tests__/bounce.service.spec.ts

```diff
@@ -138,9 +145,11 @@ describe('updateBounces', () => {
       email,
       hasBounced: false,
     }))
-    expect(mockLogger.info).toHaveBeenCalledWith(
-      JSON.parse(notification.Message),
-    )
+    expect(mockLogger.info.mock.calls[0][0]).toMatchObject({
```

Comment:
first time looking at these tests. Now I'm disappointed in myself for not mocking the logger as part of my tests :/

Reply:
lol no la it's just that the log statement is particularly important here, because our alarms depend on it